navit/graphics.c

```diff
@@ -1911,12 +1911,13 @@ static void draw_init_ctx(struct draw_polyline_context *ctx, int maxpoints) {
 
 static void graphics_draw_polyline_as_polygon(struct graphics *gra, struct graphics_gc *gc,
         struct point *pnt, int count, int *width) {
+    if (count < 2)
```

Comment:
I wouldn't do that change.
A: this is forbidden in pre C99 C.
B: Variables in question are initialized static and will be copied on stack as a block by compiler most probably.
So I'd propose either leave this as is, or else put whole function in "else" block instead. Then the variables are in another context. But changes are high that even then they are copied as block on stack.

Reply:
Alright, closing then.